### ISSUE TITLE
How to filter transactions to only obtain transactions that took place after a certain date?

### ISSUE THREAD

**Issue by andreagiovanesorare** [sorare/api#324]
Is there a way to filter transactions by date directly from GraphQl to avoid having to look at them all every time?

**Comment by finaldzn** [comment]
You should put them all in a sql database and then search for it from there

**Comment by andreagiovanesorare** [comment]
Yes, I know that I can filter from a sql database. My question is how to filter directly from GraphQl Sorare API. Otherwise at each time I have to pull all the transactions

**Comment by andreagiovanesorare** [comment]
If I have saved inside my database all the transactions for mbappé until yesterday 26-10-2022 16:00, I want to get from GraphQl only the new transaction for mbappé, so started from 26-10-2022 16:00.
Is there a way to do this?

**Comment by andreagiovanesorare** [comment]
Ok, these are the worst APIs ever seen
